MTA-6893 Document Hub OIDC authentication (#382)
migtools/mta-documentation: assemblies/mta-install-title/assembly_ldap-authentication.adoc

:_newdoc-version: 2.18.5
:_template-generated: 2025-07-31
:_mod-docs-content-type: ASSEMBLY

ifdef::context[:parent-context-of-ldap-authentication: {context}]

ifndef::context[]
[id="ldap-authentication"]
endif::[]
ifdef::context[]
[id="ldap-authentication_{context}"]
endif::[]
= LDAP authentication

:context: ldap-authentication

[role="_abstract"]

> Manage users by organization hierarchy, apply centralized identity management, and scale role assignment, by integrating the Hub OpenID Connect (OIDC) provider with Lightweight Directory Access Protocol (LDAP)-based authentication.

Reviewer: ```suggestion
You can manage users by organization hierarchy, apply centralized identity management, and scale role assignment by integrating the Hub OpenID Connect (OIDC) provider with LDAP-based authentication.
```
Second person
There is an unnecessary comma before the prepositional phrase `scale role assignment`
